distribution/docs/src/main/resources/content/_metadataAttributes/isr-attributes-table.adoc

```diff
@@ -59,8 +59,8 @@
 
 |isr.niirs
 |The quality or detail level of an image. (National Imagery Interpretability Rating Scale)
-|Integer
-|0-9
+|Float
+|\#.# where # is a value between 0-9
```

Comment:
❓ What does the beginning `\` mean here?

Reply:
It escapes `#` in the whole line.